<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>password</class>
 <widget class="QDialog" name="password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="change">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>100</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
color : black
}</string>
   </property>
   <property name="text">
    <string>old password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="conf_mdp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>190</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
color : black
}</string>
   </property>
   <property name="text">
    <string>new password confirmation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
color : black
}</string>
   </property>
   <property name="text">
    <string>new password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>59</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font-weight: bold;
font-size: 15px;
color : black
}</string>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mdp">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_employer">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ancien_mdp">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
